docs/integrations/airflow.md

```diff
@@ -155,3 +155,7 @@ default_scheduler:
   type: mwaa
   environment: <The MWAA Environment Name>
 ```
+
+### Yandex Cloud
+
+Yandex Cloud offers [managed Airflow instances](https://yandex.cloud/en/services/managed-airflow), and SQLMesh fully supports Airflow hosted by Yandex Cloud.
```

Comment:
Sorry about the back and forth but it'd be helpful to mention which scheduler type to specify to use this. Eg. here's how we document this for the Cloud Composer: https://github.com/TobikoData/sqlmesh/blob/13cf28ab60972ef043fbe6c65ddc11908f949631/docs/reference/configuration.md#cloud-composer. We can then include a link to the config reference here.

Reply:
Thank you! Somehow I missed the part about the configuration :D

I added information about the configuration in the latest commit and slightly adjusted the wording in `docs/integrations/airflow.md`